Cargo.toml

```diff
@@ -21,7 +21,8 @@ exclude = [
 	"tests/contracts",
 ]
 members = [
-	"integration-tests",
+	# TODO: add back in once the 2412 release is fixed.
```

Comment:
Could we link the issue that needs fixing ?

Reply:
fixed in the #425